nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/common-ui.css

```diff
@@ -150,9 +150,8 @@ ul.links span.header-link-over {
 }
 
 .blank, .unset, .sensitive {
```

Comment:
@scottyaslan Where `.unset` is being applied here, it needs `font-size: 13px` and `padding-top: 2px`
Not sure all places the other classes are referenced...